client/src/components/AuthenticatedRoute.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+/**
+ * This Component is a subclass of Route. It contains code for determining
+ * whether or not a user is logged in and only allows that user to access
+ * his/her desired Route if he/she is logged in.
+ * @param {Object|null} props - The data passed to this Component when it was
+ * created or by its parent Component.
+ * @return {Component|null|false} - HTML representation of this Component
+ * for DOM rendering or null/false if nothing is to be rendered.
+ */
+export default function AuthenticatedRoute(props) {
+  const token = window.localStorage.getItem('token');
+  if (token !== '' && token !== null) {
```

Comment:
Hmmm. Now that you've metioned it, I can think of another approach. I will look for a secure way to decode the token found. If I don't find one, then I toughen the logic in AuthenticatedRoute.